bin/ncar_run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This script should be executed under /path_to_samurai_root/run directory
+
+EXE="../build/release/bin/samurai"
```

Comment:
This script still appears to make assumptions about where it is run.  I find this problematic.  We need a cleaner way that does not break when Samurai is not executed from the bin directory

Reply:
So maybe you prefer an absolute rather than relative path here?